src/django_management/urls.py

```diff
@@ -20,6 +20,8 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('allauth.urls')),
```

Comment:
it looks like this is causing the error - when we navigate to `localhost/`, it auto redirects to `allauth.urls` which doesn't exist as a module in the code.

I also imagine an error would occur since we have to paths to the same path (both to `''` or the `localhost/` site.

Reply:
Please go migrations before starting the server.

Reply:
Please check the PR dependencies in the description. I will go over this in next meeting.

Reply:
Same error on `python src/manage.py migrate` and `python src/manage.py makemigrations`

ModuleNotFoundError: No module named 'allauth'

Reply:
I've updated requirements.txt. Please run `pip install -r requirements.txt` in your virtualenv.

Reply:
nice that worked ! great work!! 

let's hold off on merging this until we can get the google auth setup on our locals and potentially get that into a GitHub environment variable.